### PR TITLE
Added program List_lab

### PR DESCRIPTION
**Pull request by Chieu1**

### CODE REVIEW

File: students/chieu_quach/lesson03/List_lab.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+#  Author      - Chieu Quach
+#  Assignment  - Lesson 3
+#  Exercise    - List Lab
+
+
+def loop_func(lenfruit,typeinput,fruit):
+      
+      
+      i = 0
```
Comment (natasha-aleksandrova) [inline students/chieu_quach/lesson03/List_lab.py]: your indentation is wrong - should be exactly 4 spaces

File: students/chieu_quach/lesson03/slicing_lab.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+     # With every other item removed
+     # The line end = " " - means to print output in the same line
+     lstr = len(seq)
+     n = 0
+     for n in range(0,lstr):
+         v = n % 2
+         if v == 1:
+            print(seq[n], end = " ")
+            return seq(n)
```
Comment (natasha-aleksandrova) [inline students/chieu_quach/lesson03/slicing_lab.py]: this is not using slicing - please fix

File: students/chieu_quach/lesson03/slicing_lab.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+a_string = "this is a string"
+a_tuple = (2, 54, 13, 12, 5, 32)
+exchange_first_last(a_string)
+
+swap_element(a_string)
+swap_element(a_tuple)
+assert swap_element(a_string) == "gnirts a si siht"
+assert swap_element(a_tuple) == (32, 5, 12, 13, 54, 2)
+assert exchange_first_last(a_string) == "ghis is a strint"
+assert exchange_first_last(a_tuple) == (32, 54, 13, 12, 5, 2)
```
Comment (natasha-aleksandrova) [inline students/chieu_quach/lesson03/slicing_lab.py]: there should be 5 functions - you only have 3.

File: students/chieu_quach/lesson03/List_lab.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+      # Use tools like apppend and insert to add user input
+      # item to fruit table
+      
+      # fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+      print (fruit)
+      # Append fruit to the list
+      response = input("Enter a fruit to be added ")
+      fruit.append(response)
+      lenfruit = len(fruit)
+      loop_func(lenfruit,"append",fruit)
```
Comment (natasha-aleksandrova) [inline students/chieu_quach/lesson03/List_lab.py]: what is a `loop_func` ? it seems to be doing a lot of things - best to have your function do one thing and be clear on what it's doing.

**Comment by Chieu1** [comment]
Natasha,

I will review and correct the mistakes that you mentioned.

Thanks,

Chieu

**Comment by Chieu1** [comment]
Natasha,

I made the changes on the two programs and uploaded them to Github. Please take a look.

Thanks,

Chieu

**Pull request merged**